docs/pages/index.mdx

```diff
@@ -57,9 +63,223 @@ endpoints, infrastructure assets, and AI agents.
 You can quickly see how Teleport works by following our [Get
 Started](./get-started.mdx) guide to enroll your first resource with Teleport.
 
-(!docs/pages/includes/homepage/products.mdx!)
+<Products
+  productCategories={[
+    {
+      id: 'zero-trust-access',
+      title: 'Zero Trust Access',
+      href: './zero-trust-access/',
+      description: 'Easy access to all your infrastructure on a foundation of cryptographic identity',
+      iconColor: '#EEEAFA',
+      iconComponent: zeroTrustSvg,
+      features: [
+        {
+          title: 'Enroll and Protect Your Infrastructure',
+          description: 'Apps, servers, databases, Kubernetes, desktops, & more',
+          href: './enroll-resources/'
+        },
+        {
+          title: 'VNet: Build without VPNs',
+          description: 'Secure app & SSH access with no VPNs or proxies',
+          href: './enroll-resources/application-access/guides/vnet/'
+        },
+        {
+          title: 'Secure MCP (Protect the Vibes)',
+          description: 'Secure MCP integration with granular audit trail',
+          href: './connect-your-client/model-context-protocol/'
+        },
+        {
+          title: 'Role-Based Access Control (RBAC)',
+          description: 'Govern infrastructure access with granular permissions',
+          href: './admin-guides/access-controls/'
+        },
+        {
+          title: 'Passwordless Authentication',
+          description: 'Log in securely using biometrics',
+          href: './admin-guides/access-controls/guides/passwordless'
+        },
+        {
+          title: 'Integrate with SSO Providers',
+          description: 'Connect Okta, Entra ID, Google, and more',
+          href: './admin-guides/access-controls/sso/'
+        },
+        {
+          title: 'Structured Audit Export',
+          description: 'Forward audit logs to SIEMs like Splunk and Datadog',
+          href: './admin-guides/management/export-audit-events/'
+        },
+        {
+          title: 'Identity-Based Audit Events',
+          description: 'Detailed audit logs for every user action',
+          href: './reference/monitoring/audit/'
+        },
+        {
+          title: 'Session Recording and Playback',
+          description: 'Record a detailed review of what took place',
+          href: './reference/agent-services/desktop-access-reference/sessions/'
```

Comment:
This should be linking to the more general session recordings page, not a desktop access page.